src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/sge/SGEJobManager.groovy

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2017 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.execution.jobs.cluster.sge
+
+import de.dkfz.roddy.StringConstants
+import de.dkfz.roddy.config.ResourceSet
+import de.dkfz.roddy.execution.BEExecutionService
+import de.dkfz.roddy.execution.jobs.BEJob
+import de.dkfz.roddy.execution.jobs.GenericJobInfo
+import de.dkfz.roddy.execution.jobs.JobManagerCreationParameters
+import de.dkfz.roddy.execution.jobs.ProcessingCommands
+import de.dkfz.roddy.execution.jobs.cluster.pbs.PBSJobManager
+import de.dkfz.roddy.execution.jobs.cluster.pbs.PBSResourceProcessingCommand
+import de.dkfz.roddy.tools.BufferUnit
+import groovy.transform.CompileStatic
+import sun.reflect.generics.reflectiveObjects.NotImplementedException
+
+/**
+ * Created by michael on 20.05.14.
+ */
+@CompileStatic
+class SGEJobManager extends PBSJobManager {
+    SGEJobManager(BEExecutionService executionService, JobManagerCreationParameters parms) {
+        super(executionService, parms)
+    }
+
+    @Override
+    SGECommand createCommand(GenericJobInfo jobInfo) {
+        throw new NotImplementedException()
+    }
+
+    SGECommand createCommand(BEJob job, List<ProcessingCommands> processingCommands, String command, Map<String, String> parameters, Map<String, Object> tags, List<String> dependencies, File logDirectory) {
+        SGECommand sgeCommand = new SGECommand(this, job, job.jobID.toString(), processingCommands, parameters, tags, null, dependencies, command, logDirectory)
+        return sgeCommand
+    }
+
+    @Override
+    SGECommand createCommand(BEJob job, String jobName, List<ProcessingCommands> processingCommands, File tool, Map<String, String> parameters, List<String> dependencies) {
+        throw new NotImplementedException()
+    }
+
+    SGECommand createCommand(BEJob job) {
+        return new SGECommand(this, job, job.jobName, [], job.parameters, [:], [], job.dependencyIDsAsString, job.tool?.getAbsolutePath() ?: job.getToolScript(), job.getLoggingDirectory())
+    }
+//    @Override
+//    public void addSpecificSettingsToConfiguration(Configuration configuration) {
```

Comment:
Can it be removed?

Reply:
Remove this. This code does not belong into BE because scratch directories are cluster specific and should be configurable (e.g. in the new LSF cluster it will be /local/$USER). It is removed in my branch anyway.